content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,361 @@
    :align: center
-   :alt: features to enable in order to properly use removal strategies.
+   :alt: The detailed operations for the transfer shows the lots to be removed.
+
+Using removal strategies
+========================
+
+To differentiate some units of products from others, the units need to be tracked, either by
+:guilabel:`Lot` or by :guilabel:`Serial Number`. To do so, go to :menuselection:`Inventory -->
+Configuration --> Settings`. Then, activate the :guilabel:`Storage Locations`, :guilabel:`Multi-Step
+Routes`, and :guilabel:`Lots & Serial Numbers` settings. Click :guilabel:`Save` to save changes.
+
+.. image:: removal/traceability.png
+   :align: center
+   :alt: :alt: Traceability settings.
+
+.. image:: removal/warehouse-settings.png
+   :align: center
+   :alt: :alt: Warehouse settings.
 
 .. note::
-   To work with the *FEFO* strategy, activate the *Expiration Dates* feature.
+   To use the :abbr:`FEFO (First Expired, First Out)` removal strategy, the :guilabel:`expiration
+   dates` setting needs to be activated as well. To enable this, go to :menuselection:`Inventory app
+   --> Configuration --> Settings`, scroll down to the :guilabel:`Traceability` section, and click
+   the checkbox next to :guilabel:`Expiration Dates`. Remember to click :guilabel:`Save` to save all
+   changes.
+
+Now, specific removal strategies can be defined on product categories. To do this, go to
+:menuselection:`Inventory app --> Configuration --> Product Categories`, and choose a product
+category to define the removal strategy on. In the :guilabel:`Force Removal Strategy` field, choose
+a removal strategy.
+
+.. image:: removal/product-category-removal.png
+   :align: center
+   :alt: :alt: Removal strategy on a product category.
 
-Next, you need to define your removal strategy, on *Product Categories* via
-:menuselection:`Inventory --> Configuration --> Product Categories`.
+To view all products with lots/serial numbers assigned to them, navigate to
+:menuselection:`Inventory app --> Products  --> Lots/Serial Numbers`. This reveals a page with
+drop-down menus of all products assigned lots or serial numbers, filtered by *product* by default.
+To change the category these products are filtered by, click :guilabel:`Product` (in the search bar,
+in the top right of the page) to remove the default filter, and select a new filter if desired.
 
-.. image:: removal/first-in-first-out.png
+.. image:: removal/lot-serial.png
    :align: center
-   :alt: force removal strategy set up as first in first out.
+   :alt: Click on Products, then Lots/Serial Numbers to display all the products with lots or serial
+         numbers.
+
+To view the serial numbers being selected for a sales order, go to the :guilabel:`Sales app` and
+select the sales order in question. In the sales order, click the :guilabel:`Delivery` smart button
+in the top right. In the :guilabel:`Operations` tab, click the :guilabel:`⋮≣ (Detailed Operations)`
```

Comment:
I think this is ok to use two icons to form the UI icon, although I think the one with 4 dots and 4 lines is a closer match
```suggestion
in the top right. In the :guilabel:`Operations` tab, click the :guilabel:`⦙≣ (Detailed Operations)`
```

Reply:
Thank you! I think KSC found the icon for me since I couldn't find this one on my own (I still don't know how people find icons I can't seem to find).